docs/architecture/adr-024-sign-bytes.md

```diff
@@ -208,7 +208,7 @@ func (tp *T) SignBytes() []byte {
 
 ## Status
 
-DRAFT
+Partially Accepted
```

Comment:
Is it obvious in the ADR what parts are accepted and what parts are not

Reply:
marked it as accepted, with the migration to grpc much of it is obsolete